README.md

```diff
@@ -43,7 +43,7 @@
 2. Open ports 443, 80, 53, and 51821-51830/udp on the VM firewall and in cloud security settings.
 3. Run the script **(see below for optional configurations)**:
 
-`sudo wget -qO - https://raw.githubusercontent.com/gravitl/netmaker/master/scripts/nm-quick.sh | bash`
```

Comment:
Sudo should be applied to bash command and not the wget command.
This was tested on ubuntu 20.04